### ISSUE TITLE
Minimum Color Contrast Ratio

### ISSUE THREAD

**Issue by Dayvit** [SoftProjTeam3-1/SENG3150A#259]
Description:
Fix any of the following: Element has insufficient color contrast of 2.34 (foreground color: #ffffff, background color: #ff8904, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1. (white and orange). This was shown in Axe accessibility testing.
There are 16 total issues of color ratio listed in acceptance criteria. Some the same colours just different components.
https://axe.deque.com/axe-watcher/issues/SENG3150A/41dedbb2-8d5c-4f85-aa8c-825c524fe8e7/sessions/db14c454-ef01-49f7-ac2c-39cd9db4a6de/9dab06ac-2f47-4137-8eae-63d51bd1ca16/color-contrast/10?settings_hash=6be64c196d517f14eb90e9fd00a0c7f2&issues_over_a11y_threshold=17&issue_filters=
Acceptance Criteria
- #ffffff White and #ff8904 Orange
- #ff8904 Orange and #fcebe9 Salmon Pink
- #fffdfd Redish White and #fcebe9 Salmon Pink
- #fadbd8 Another Shade of Redish white and #f8cdc8 A darker shade
- #fbe0de Another Redish white and #fad8d4 Redish white
- #fcfefc A off White and #e3f7e4 Greenish white
- #ffffff white and #07bc0c green
